ci/docker-compose.yml

```diff
@@ -146,6 +146,20 @@ services:
     entrypoint: sleep
     command: infinity
 
+  jwks_py:
+    image: python:3
+    ports:
+      - 8090:8090
+    entrypoint:
+      - bash
+      - -c
+    command:
+      - |
+        pip install jwcrypto
+        python /usr/src/jwks/jwks_server.py -p 8090
+    volumes:
+      - ./jwt/jwks_server.py:/usr/src/jwks/jwks_server.py
```

Comment:
The ./ is needed?

Reply:
Yes. It means relative to the current directory path.